libs/remix-ui/settings/src/lib/remix-ui-settings.tsx

```diff
@@ -114,47 +123,53 @@ export const RemixUiSettings = (props: RemixUiSettingsProps) => {
     )
   }
 
-  const saveToken = () => {
-    saveTokenToast(props.config, dispatchToast, tokenValue)
+  const saveToken = (type: string) => {
+    saveTokenToast(props.config, dispatchToast, tokenValue[type], labels[type].key)
   }
 
-  const removeToken = () => {
-    setTokenValue('')
-    removeTokenToast(props.config, dispatchToast)
+  const removeToken = (type: string) => {
```

Comment:
on clicking `Remove` for Etherscan access token, it says, `Access token removed` but doesn't remove it from textbox.

If `access token` is updated to `api key` for Etherscan, please update the toaster text too.

https://files.gitter.im/5e451c02d73408ce4fd98526/zEWj/Screenshot-2022-03-03-at-7.44.16-PM.png